### DISCUSSION TITLE
Adventure map, indicate the number of days required to move on the last arrow of the path for each day

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3299] [Ideas]
In fheroes2, it would be nice to indicate the number of days required to move on the last arrow of the path for each day (it's displayed for travel on land and sea).
Per example, the number "3" is added to the last arrow of the third day of travel.

Screens are for explanation only.
![Fheroes2 Screen 055](https://user-images.githubusercontent.com/43583994/116444422-ff5df900-a854-11eb-897c-e04d85494ae4.png)
![Fheroes2 Screen 056](https://user-images.githubusercontent.com/43583994/116444436-038a1680-a855-11eb-99a9-873953fa6176.png)

Also, I think there should be additional cursors for travels longer than 4 days, i.e. "5", "6", etc. (currently the cursor value is limited to "4+").

**Comment by ihhub** [comment]
With our engine we can do this. @Branikolog , what's you opinion?

**Reply by Branikolog** [reply]
Definitely a great proposal.
I'll be glad to watch at least up to 7 days path calculation. As well as days showed at the last arrow of each daily route.

**Reply by LeHerosInconnu** [reply]
@ihhub and @Branikolog.

> Definitely a great proposal.

Thanks. :)

> I'll be glad to watch at least up to 7 days path calculation.

The path can sometimes be long on the ocean in some scenarios.
All cases should be covered.

In fheroes2:

![Path on ocean 001](https://user-images.githubusercontent.com/43583994/118140851-0b2aed00-b409-11eb-8b0f-fa572f191e9c.png)
![Path on ocean 002](https://user-images.githubusercontent.com/43583994/118140860-0e25dd80-b409-11eb-8fe2-d2978c43899c.png)

**Comment by LeHerosInconnu** [comment]
This is planned in my mod: https://github.com/ihhub/fheroes2/discussions/10120.
I close the discussion.
